client/src/components/AddItem.js

```diff
@@ -52,14 +54,22 @@ const useStyles = makeStyles((theme) => ({
     width: "7rem",
     marginRight: "5rem",
   },
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 120,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing(2),
+  },
+  
 }));
 
 const AddItem = () => {
   const userId = useContext(UserContext).userId;
   const [inputLink, setInputLink] = useState("");
   const [item, setItem] = useState({});
   const [popupOpen, setPopupOpen] = useState(false);
-  const [selectedListIndex, setSelectedListIndex] = useState(0);
+  const [selectedListIndex, setSelectedListIndex] = useState("");
```

Comment:
Shouldn't this be a number?

Reply:
this makes it so the default text is "Select List" rather than the first item on the list.